packages/api/src/api/ProviderApi.ts

```diff
@@ -74,4 +88,8 @@ export default class ProviderApi extends HttpClientBase {
         }
         return this.axios.post(`/v1/prosopo/provider/verify`, payload)
     }
+
+    private getNewRandomProvider(userAccount: string): Promise<ProsopoRandomProvider> {
```

Comment:
is this being called in multiple places? If not, you could inline it like the other axios calls